docker/gym-aquaticus/gym_aquaticus/envs/config.py

```diff
@@ -212,4 +212,83 @@ def __init__(self):
 
         # MOOS Timewarp is the simulation speed-up factor
         # This must match the value in your simulation script
-        self.moos_timewarp = 4.0
\ No newline at end of file
+        self.moos_timewarp = 4.0
+
+class JervisBayConfig:
+    '''
+    This is the configuration for Jervis Bay 2023.
+    '''
+    def __init__(self, script='./get_field.sh'):
+        subprocess.call(['.sh', script])
+        #Get Values from saved value
+        f = open('field.txt', 'r')
+        x = f.readlines()[:3]
+        #field_bounds = [bl, tl, tr, br]
+        field_bounds = []
+        #zone1 = [bl, tl, tr, br]
+        zone1 = []
+        #zone2 = [bl, tl, tr, br]
+        zone2 = []
+        for v in range(len(x)):
+            for i in x[v].split('{')[1].split('}')[0].split(':'):
+                vals = i.split(',')
+                if v == 0:
+                    field_bounds.append((float(vals[0]), float(vals[1])))
+                elif v == 1:
+                    zone1.append((float(vals[0]), float(vals[1])))
+                else:
+                    zone2.append((float(vals[0]), float(vals[1])))
+        f.close()
+
+        f = open('flags.txt', 'r')
+        flags = f.readlines()
+
+        bf = []
+        rf = []
+        i = 0
+        for flag in flags[:2]:
+            if i == 0:
+                bf = [float(flag.split(',')[0].split('=')[-1]), float(flag.split(',')[1].split('=')[-1].split('\"')[0])]
+            else:
+                rf = [float(flag.split(',')[0].split('=')[-1]), float(flag.split(',')[1].split('=')[-1].split('\"')[0])]
+            i += 1
+        f.close()
+
+        # Simulation environment
+        self.sim_timestep = 0.4      # moostime (sec) between steps
+        self.sim_time_limit = 180.0  # moostime (sec) before terminating episode
+        self.capture_radius = 8.0
+        self.flag_range = 10.0
+
+        # The zones and positions are generated by jerop.sh in the mission folder
+        # and stored in region_info.txt
+        # The flag positions are defined in shoreside/launch_shoreside.sh
+        # (propagated to targ_shoreside.moos) and must be matched here
+        self.blue_flag = bf#[140.0, 190.0]
+        self.red_flag =  rf#[20.0,  190.0]
+
+        self.red_zone_ul = [zone2[1][0],  zone2[1][1]]
+        self.red_zone_ur = [zone2[2][0], zone2[2][1]]
+        self.red_zone_ll = [zone2[0][0], zone2[0][1]]
+        self.red_zone_lr = [zone2[3][0], zone2[3][1]]
+
+        self.blue_zone_ul = [zone1[1][0],  zone1[1][1]]
+        self.blue_zone_ur = [zone1[2][0],  zone1[2][1]]
+        self.blue_zone_ll = [zone2[0][0],  zone2[0][1]]
+        self.blue_zone_lr = [zone2[3][0],  zone2[3][1]]
+
+        self.boundary_ul = self.red_zone_ul
+        self.boundary_ur = self.blue_zone_ur
+        self.boundary_ll = self.red_zone_ll
+        self.boundary_lr = self.blue_zone_lr
+
+        self.scrimmage_pnts = [self.red_zone_lr, self.red_zone_ur]
```

Comment:
Note that the scrimmage points depend on the orientation of the field. They're not always the same... This is also true for the boundary points. These scrimmage points are for a horizontal field. If the field is oriented differently it could be `[self.red_zone_ll, self.red_zone_lr]`

Reply:
This will mess up the observations because it won't know when it's on the correct side.